docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,45 @@
+# Configuring interactive sessions
+
+## Inactive interactive sessions cleanup
+
+### Automatic closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed. This helps optimize resource utilization and ensure that unused sessions
+are not consuming unnecessary resources.
+
+The automatic closure of interactive sessions in REANA can be customized by configuring
+[Helm values](https://github.com/reanahub/reana/tree/master/helm/reana):
+
+- `interactive_sessions.maximum_inactivity_period`:
+  This option allows you to set a limit (in days) for the maximum inactivity period of interactive sessions after which they will be closed.
+  If this option is set to `forever`, interactive sessions will be kept open indefinitely unless manually closed.
```

Comment:
... will be kept open indefinitely until they are closed manually.
(This the default value.)